apps/mobile/src/components/OfferDetailScreen/components/OfferInfo.tsx

```diff
@@ -83,14 +84,21 @@ function OfferInfo({
     if (!text.trim()) return
     void pipe(
       submitRequest({text, originOffer: offer}),
-      TO.map((chat) => {
-        navigation.replace('ChatDetail', {
-          chatId: chat.id,
-          inboxKey: chat.inbox.privateKey.publicKeyPemBase64,
-        })
-      })
+      TE.match(
+        (e) => {
+          if (e._tag === 'ReceiverOfferInboxDoesNotExistError') {
```

Comment:
Move flagging here to make it more readable and apparent why there is call to goBack func